examples/light-switch-app/silabs/include/LightSwitchMgr.h

```diff
@@ -23,6 +23,9 @@
 #include <lib/core/CHIPError.h>
 #include <lib/support/CodeUtils.h>
 
+#define MIN_LEVEL 0
+#define MAX_LEVEL 254
```

Comment:
This doesn't work within the cluster functions. Level Control has two attributes that defines this. We cannot redefine arbitrary values.

Reply:
I did not add any attributes of Level Control cluster in the zap file. Since Level control cluster is added as a client(similar to On/Off cluster), i followed the same way to implement this.

Reply:
If the level is changed by another controller, the currentLevel attribute will store that value. I;ve declared these 2 variables just for the MIN and MAX values of level as i did not add any attributes in the level control cluster. 
Let me know if i should remove this and add the cluster attributes and use them.

Reply:
My point is that it doesn't make sense to hardcode these values since any light device can have a different value. You are assuming configuration and behaviors which will not work outside of a bulb with those configurations.